code/model/modelrender.cpp

```diff
@@ -2213,7 +2213,7 @@ void model_queue_render_thrusters(model_render_params *interp, polymodel *pm, in
 			if (thruster_info.tertiary_glow_bitmap >= 0) {
 				p.screen.xyw.w -= w;
 				p.r = p.g = p.b = p.a = (ubyte)(255.0f * fog_int);
-				batching_add_volume_bitmap_rotated(thruster_info.tertiary_glow_bitmap, &p, magnitude * 4, w * 0.6f * thruster_info.tertiary_glow_rad_factor, fog_int, -(D > 0) ? D : -D);
+				batching_add_volume_bitmap_rotated(thruster_info.tertiary_glow_bitmap, &p, magnitude * 4, w * 0.6f * thruster_info.tertiary_glow_rad_factor, fog_int, -w*0.5f);
```

Comment:
Apparently, the missing D is causing problems for the CI....

Reply:
Yeah, I believe `D` becomes unused, I'll just remove it.